dashboard/app/models/levels/pythonlab.rb

```diff
@@ -48,4 +50,16 @@ def self.create_from_level_builder(params, level_params)
   def uses_lab2?
     true
   end
+
+  # Ensure that if this is a multiple choice predict level, there is at least one correct answer
```

Comment:
Should we check this on the front end too, before saving?

Reply:
This will cause an error message to show up on for levelbuilders when they click save. I didn't want to have annoying red errors while they were editing, and it's not a pattern we currently follow.